packages/hardhat-plugin-ethers/src/utils/error-deployment-result-to-exception-message.ts

```diff
@@ -0,0 +1,141 @@
+import {
```

Comment:
This util is the only one currently being shared between both ethers and viem plugins. Is this something we should maybe export from the base plugin?

Reply:
I have exported it from the base plugin.
On the adding abstractions vs duplication scale I tend towards duplication until we know we really have the abstraction; however I think if `error-deployment-result-to-exception-message` changed in one without changing in another that would be something that right now violates the design. So I think we share it up from `hardhat-plugin`.